exercises/E-conditionals/exercise.js

```diff
@@ -6,8 +6,18 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-let name = "Daniel";
-let danielsRole = "mentor";
+function Role(name, danielsRole) {
+ 
+  if (name == "Daniel" && danielsRole == "mentor"){
+    return "Hi, I'm Daniel, I'm a mentor.";
+  }if (danielsRole == "student"){
+    return "Hi, I'm Daniel, I'm a student.";
```

Comment:
This is actually related to the alternative way of structuring functions. If, for example, I passed `Role("Daniel", "not mentor"), this function would return `undefined`.